quinn-proto/src/tests/mod.rs

```diff
@@ -2070,39 +2114,6 @@ fn connect_runs_mtud_again_after_600_seconds() {
     assert_eq!(pair.server_conn_mut(server_ch).path_mtu(), 1452);
 }
 
-#[test]
-fn packet_loss_and_retry_too_low_mtu() {
```

Comment:
Why is this being removed?

Reply:
This test is mostly a sanity check I added when I was figuring out how everything worked. Now we have many tests that trigger packet loss and recovery, I deemed it unnecessary to keep it. I'll happily resurrect it, though, if you think it's worthwhile.

Reply:
Makes sense; I'm happy to leave it out.